### ISSUE TITLE
Missing floor on vehicle select

### ISSUE THREAD

**Issue by Tharmon** [egingric/2013-Racing-Game#475]
Everything floats!

**Comment by egingric** [comment]
Yes, that's "as designed" on my side. I had to get rid of the ground because of the directional light hitting it. You will need to add your own ground in please.
